api.js

```diff
@@ -4,6 +4,18 @@ const ncNews = axios.create({baseURL: "https://nc-news-97rk.onrender.com/api"})
 
 export function getArticles(){
     return ncNews.get('/articles').then(({data})=>{
-        return data
+        return data;
+    })
+}
+
+export function getArticleById(articleId){
+    return ncNews.get(`/articles/${articleId}`).then(({data})=>{
+        return data;
+    })
+}
+
+export function getCommentsByArticleId(articleId){
+    return ncNews.get(`/articles/${articleId}/comments`).then(({data})=>{
+        return data;
     })
```

Comment:
Consider checking out how an [axios baseURL](https://apidog.com/blog/axios-baseurl/) works, if you implemented it here, it'd really streamline your requests